frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipelines-as-code.feature

```diff
@@ -101,7 +101,22 @@ Feature: Perform Actions on repository
 
 
         @regression @to-do
-        Scenario Outline: Delete the repository from the Repository details page: P-11-TC09
+        Scenario Outline: Pipeline Run Details page for the repository: P-11-TC09
+            Given pipeline run is displayed for "<repository_name>"
+              And user is at the repositories page
+             When user clicks Last Run value of repository "<repository_name>"
+             Then user will be redirected to Pipeline Run Details page
+              And user is able to see Details, YAML, TaskRuns, Logs and Events tabs
+              And Details tab is displayed with field names Name, Namespace, Labels, Annotations, Created At, Owner, Status, Pipeline, Repository, Branch, commit id and Event type
```

Comment:
For a pipelineRun, I think only these four fields (Repository, Branch, commit id and Event type) are related to Repository info and rest is common for any resource.

Suggestions:
1. Update the scenario to look for these four fields.
2. commit Id will be a external link, and on hover you will get the tooltip of the commit message. Shall we include this ?